etc/cewrapper/compilers-and-tools.json

```diff
@@ -0,0 +1,9 @@
+{
+    "allowed_paths": [
+        {
+            "path": "Z:\\",
+            "rw": false
+        }
+    ],
+    "allowed_registry": []
+}
```

Comment:
Some missing newlines at ends of files here (only noticed at github makes a meal out of them!)